src/postamble.js

```diff
@@ -420,6 +441,7 @@ function exit(status, implicit) {
 #endif
   }
 
+  postRun();
```

Comment:
This looks surprising, why is it needed? Can't we end up calling it twice?

Reply:
Sorry these are used in args_sizes_get and args_get, which I forgot to include.    Should be there now.

Reply:
The difference here is that `_start` doesn't return a value, so when the program exits normally it calls "exit(rtncode)" from `crt1.c`.   This means that we end up here in `exit()` with implicit = false, more often then we would with the old code.

But actually this was only needed when I was trying to run all tests with STANDALONE_WASM.  For the current subset this isn't needed.  I can delete.